JarrydWorland/2021_Capstone_GP2_Team
# Enemy charger

## Description
This PR adds the new charger enemy and relevant assets.

## How to test
Place the charger enemy within one of the room prefabs so that it spawns naturally, then find a room containing the enemy and ensure it behaves properly. Like with other enemies, placing the enemy manually while the game is running may result in unexpected behavior.
